.pre-commit-config.yaml

```diff
@@ -66,6 +67,23 @@ repos:
     rev: v2.5.0
     hooks:
       - id: setup-cfg-fmt
+  - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
This is archived, it'd be nice to find an alternative in the future. Anyway, not a big deal and it only affects the documentation app.